examples/08_interpolation_join.py

```diff
@@ -0,0 +1,179 @@
+"""
+Interpolation join: infer missing rows when joining two tables
+==============================================================
+
+We illustrate the ``InterpolationJoiner``, which is a type of join where values from the second table are inferred with machine-learning, rather than looked up in the table.
+It is useful when exact matches are not available but we have rows that are close enough to make an educated guess -- in this sense it is a generalization of a ``fuzzy_join``.
```

Comment:
Same for fuzzy_join. If the links need to be repeatedly added (as it seems to me) look at the other examples for a way to link them all to one.
```suggestion
It is useful when exact matches are not available but we have rows that are close enough to make an educated guess -- in this sense it is a generalization of a :func:`skrub.fuzzy_join`.
```